cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml

```
@@ -326,3 +326,27 @@ spec:
       for: 0s
       labels:
         severity: critical
+  - name: pool-quota.rules
+    rules:
+    - alert: PoolQuotaBytesNearExhaustion
```

Considering support of other Storage Providers, we can add Ceph to the name of the alert.
```suggestion
    - alert: CephPoolQuotaBytesNearExhaustion
```